### DISCUSSION TITLE
Chainladder vs Over Dispersed Poisson glm

### DISCUSSION THREAD

**Discussion by karlzw** [casact/chainladder-python#423] [General]
According to academic literature the ibnr for the chainladder and the ODP should be the same. 
When using the TweedieGLM this is somewhat the case for the genins sample data set.
However on another dataset I have am getting completely different reserves.
What could be the issue?

**Comment by jbogaardt** [comment]
Similar to last discussion, its difficult to say without a [reprex](https://reprex.tidyverse.org/).  I would say that any violation of the Tweedie assumptions would cause a difference. All incremental values of triangle must be greater than 0 so that the log of the incrementals can be computed.  Further, the Tweedie model must use a poisson error with log link function and the deterministic chainladder would need to use the volume-weighted all period average. I think in both cases, `TweedieGLM` and `Chainladder` without further configuration should default to this basis.
